### DISCUSSION TITLE
Path stroke with

### DISCUSSION THREAD

**Discussion by arcdiablo** [keiwando/toonsquid#263] [General]
Im using a lot the path tool , but every new path I make have to change the with stroke , it return always to size 40 , I missing something  ?  Thanks

**Comment by keiwando** [comment]
With the 1.9 update, the path tool now remembers the recent stroke width and some other layer settings.

**Reply by arcdiablo** [reply]
thank,s for responding, I will update the app,

**Reply by DWrightAnimation** [reply]
I do have one complaint about this though, when I use the shapes in the path tool, it automatically uses 40 width stroke, and usually doesn't start with the stroke.
